### ISSUE TITLE
Multipart upload bug

### ISSUE THREAD

**Issue by xujq1** [actix/actix-web#876]
I tried the multipart in the latest example, I found a bug:

1. when I uploaded a sql file in firefox, the actix web will run infinitely, I checked the uploaded file, there is a empty form boundry int the end of file.
   but when I uploaded the same file for a php server ,it works fine
2. when I uploaded the same file in chromium , actix web works fine

I think this bug triggered by firefox and actix multipart

my env
```shell
os:  arch linux x64
browers: firefox 67 and chromium 74
rust:  1.35.0, stable
```
this is the file that trigger the bug
[file-trigger-bug.txt](https://github.com/actix/actix-web/files/3239774/file-trigger-bug.txt)

**Comment by fafhrd91** [comment]
fixed in master, actix-multipart 0.1.2 is released as well.

**Comment by xujq1** [comment]
thanks for fixing
